demosplan/DemosPlanCoreBundle/Entity/Procedure/ProcedurePerson.php

```diff
@@ -183,6 +183,13 @@ public function setCity(?string $city): ProcedurePerson
         return $this;
     }
 
+    public function getPostalCodeWithCity(bool $withCity= false): ?string
```

Comment:
`$withCity` not needed

Reply:
my bad sorry, fixed here https://github.com/demos-europe/demosplan-core/pull/2069/commits/a02e5de282d3d273f35e5a6196d32f7fc6baa7c8